extensions/amp-analytics/linker-id-receiving.md

```diff
@@ -53,3 +53,41 @@ You can find all the code for our implementaion in the [linker.js](./0.1/linker.
 
 #### Test Cases
 If you are implementing your own logic to verify these checksums, please ensure that you are using our test vectors located in [test-linker.js](./0.1/test/test-linker.js)
+
+
+### Receiving Linker Params on AMP Pages
+
+The `<amp-analytics>`'s `cookies` feature can be used to extract information from the document url and write to the origin domain.
+
+#### Configuration
+
+The `cookies` example configuration looks like
+```
+'cookies': {
+  'enabled': true,
+  'cookieName1': {
+    'value': 'QUERY_PARAM(example)',
+  },
+  'cookieName2': {
+    'value': 'LINKER_PARAM(exampleName, exampleValue)',
+  },
+}
+```
+
+The `enabled` value can be used to override default vendor settings.
+
+##### Cookie Names
+Each key value within the `cookies` config object defines the cookie key, where its value needs to be an object.
```

Comment:
`Each key within the `cookies` config object defines the cookie name. It's value needs to be an object containing a single key value pair. That key should be 'value', and its value should be the macro that determines the information stored in the cookie.`

This is a really hard sentence because the keys name is `value`. Hopefully someone else will help improve :)